build.sbt

```diff
@@ -10,7 +10,7 @@ scalaVersion := "2.11.8"
 
 lazy val versionSettings = Seq(
   // Upcoming release, or current if we're on the master branch
-  git.baseVersion := "0.8",
+  git.baseVersion := "8",
```

Comment:
As much as I was an advocate for this on Cromwell I was just wondering about this for our other tools, I came up with arguments on both sides. We should discuss.